routing/routing.go

```diff
@@ -374,6 +376,21 @@ func (r *Routing) startReceivingUpdates(o Options) {
 	}()
 }
 
+// ReturnCert matches an incoming TLS client to a TLS certificate
+//
+// TODO: Refactor 
+func (r *Routing) ReturnCert(helloInfo *tls.ClientHelloInfo) (*tls.Certificate, error) {
+	rt := r.routeTable.Load().(*routeTable)
+	for _, r := range rt.validRoutes {
+		compiledRxs := make(map[string]*regexp.Regexp)
```

Comment:
This logic could probably go in the [routing/matcher.go](https://github.com/zalando/skipper/blob/a3a29f87a6eaed3076091f556d416a67e29b25a8/routing/matcher.go) file

Reply:
Resolving this since not using routes to determine TLS certificates